pkg/scheduler/framework/plugins/defaultpreemption/default_preemption_test.go

```diff
@@ -927,6 +927,40 @@ func TestPickOneNodeForPreemption(t *testing.T) {
 	}
 }
 
+func TestPodEligibleToPreemptOthers(t *testing.T) {
```

Comment:
Can you refer to the format of #92571 to rewrite this test a bit?  In particular, use {Node/Pod}Wrapper to build the API objects.

Reply:
got it, will do after #92571 merge.

Reply:
#92571 just got merged. Please rebase to check out the latest format.